<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Robert M. Hechler </title> <meta name="author" content="Robert M. Hechler"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%90%9F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://roberthechler.github.io/publications/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Robert</span> M. Hechler </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects </a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">research </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="semeniuk_increasing_2024" class="col-sm-8"> <div class="title">On increasing equity and inclusion of early-career professionals for conferences and conference networking in Canadian fisheries and aquatic science societies</div> <div class="author"> Christina A.D. Semeniuk, Kathleen D.W. Church, Felix Eissenhauer, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Jaime Grimm, Robert M. Hechler, Bradley E. Howell, Silviya V. Ivanova, Sandra Klemet-N’Guessan, Christine L. Madliger, Jessica Reid, Kendra A. Thompson-Kumar, Ivan Arismendi, Brooke E. Penaluna, Andrea E. Kirkwood, Catherine M. Febria' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>Canadian Journal of Fisheries and Aquatic Sciences</em>, Nov 2024 </div> <div class="periodical"> *Authors ordered by career stage &amp; alphabetically </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1139/cjfas-2024-0033" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>As early-career professionals (ECPs) navigate their education and professional development in the aquatic sciences, many seek to build a network to help guide their entrance into the field. As influential organizations, scientific societies play a vital role through hosted conferences, where ECPs can meet and share ideas with others, and find mentors to facilitate their colleagues’ journey within the profession. However, not all ECPs are the same, and those from marginalized backgrounds face unique challenges. Here, we provide a perspective on ways scientific societies can ensure all members are provided with equitable opportunity to discover, access, and build career-defining networks at conference events, including the critical role of mentors in navigating obstacles to success. Our recommendations originate from an early-career networking workshop in 2022 at a Canadian fisheries and aquatic sciences conference. The day-long hybrid event comprised interactive activities and discussions on how societies and their conferences can foster and promote inclusive networking for all, including suggestions on maximizing inclusivity for online attendees. This perspective serves as a call to action for scientific societies and senior-career professionals to meaningfully engage with ECPs and marginalized members to promote transformative science.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="hug_roadmap_2024" class="col-sm-8"> <div class="title">A roadmap for fair reuse of public microbiome data</div> <div class="author"> Laura A. Hug, Roland Hatzenpichler, Cristina Moraru, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'André Soares, Folker Meyer, Anke Heyder, The Data Reuse Consortium (incl. Robert M. Hechler and 172 others), Alexander J. Probst' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em>, Jun 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2024.06.21.599698" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Science benefits from rapid, open data sharing but samples for sequencing data are expensive for data creators to acquire and process. Current guidelines for data reuse were established two decades ago, when databases were several million times smaller, necessitating an update. This article presents a roadmap to establish best practices for sequence data reuse, developed in consultation with a data consortium of 167 microbiome scientists. It introduces a Data Reuse Information tag (DRI) for public sequencing data, which will be associated with at least one Open Researcher and Contributor ID (ORCID) account. The machine-readable DRI tag indicates that the data creators prefer to be contacted prior to data reuse, and simultaneously provides data consumers with a mechanism to get in touch with the data creators. Ideally, the DRI will facilitate and foster collaborations, and serve as a guideline that can be expanded to other data types.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="hechler_revealing_2024" class="col-sm-8"> <div class="title">Revealing population demographics with environmental RNA</div> <div class="author"> <em>Robert M. Hechler</em>, and Melania E. Cristescu </div> <div class="periodical"> <em>Molecular Ecology Resources</em>, Jun 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1111/1755-0998.13951" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The analyses of environmental DNA (eDNA) and environmental RNA (eRNA) released by organisms into their surrounding environment (water, soil and air) have emerged as powerful tools for monitoring biodiversity. While eDNA has been widely adopted for the non-invasive detection of species and characterization of community composition, the utilization of eRNA is still in its infancy. Due to its functional nature, eRNA holds intriguing potential for biodiversity monitoring offering new avenues of research beyond species detection. For example, conspecifics that are almost genetically identical can exhibit distinct transcriptomic differences depending on their life stage. In this issue of Molecular Ecology Resources, Parsley and Goldberg (2024) demonstrate, through a lab-validated field study, that eRNA can be used to detect distinct life stages of amphibians. This study elegantly demonstrates that eRNA can be used not only to detect invasive or endangered species but also to reveal population demographic information important for guiding effective conservation strategies.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="hechler_environmental_2023" class="col-sm-8"> <div class="title">Environmental transcriptomics under heat stress: Can environmental RNA reveal changes in gene expression of aquatic organisms?</div> <div class="author"> <em>Robert M. Hechler</em>, Matthew C. Yates, Frédéric J. J. Chain, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Melania E. Cristescu' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Molecular Ecology</em>, Jun 2023 </div> <div class="periodical"> *Harry Smith Prize and Frontiers Planet Prize nomination </div> <div class="links"> <a class="award btn btn-sm z-depth-0" role="button">Awards info</a> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1111/mec.17152" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="award hidden d-print-inline"> <p></p> <p>Received the <strong>Harry Smith Prize</strong> 2024 for the best paper published in <em>Molecular Ecology</em> by an early career researcher, and was nominated by McGill University for the <strong>Frontiers Planet Prize</strong> 2025.</p> </div> <div class="abstract hidden"> <p>To safeguard biodiversity in a changing climate, taxonomic information about species turnover and insights into the health of organisms are required. Environmental DNA approaches are increasingly used for species identification, but cannot provide functional insights. Transcriptomic methods reveal the physiological states of macroorganisms, but are currently species-specific and require tissue sampling or animal sacrifice, making community-wide assessments challenging. Here, we test whether broad functional information (expression level of the transcribed genes) can be harnessed from environmental RNA (eRNA), which includes extra-organismal RNA from macroorganisms along with whole microorganisms. We exposed Daphnia pulex as well as phytoplankton prey and microorganism colonizers to control (20°C) and heat stress (28°C) conditions for 7 days. We sequenced eRNA from tank water (after complete removal of Daphnia) as well as RNA from Daphnia tissue, enabling comparisons of extra-organismal and organismal RNA-based gene expression profiles. Both RNA types detected similar heat stress responses of Daphnia. Using eRNA, we identified 32 Daphnia genes to be differentially expressed following heat stress. Of these, 17 were also differentially expressed and exhibited similar levels of relative expression in organismal RNA. In addition to the extra-organismal Daphnia response, eRNA detected community-wide heat stress responses consisting of distinct functional profiles and 121 differentially expressed genes across eight taxa. Our study demonstrates that environmental transcriptomics based on extra-organismal eRNA can noninvasively reveal gene expression responses of macroorganisms following environmental changes, with broad potential implications for the biomonitoring of health across the trophic chain.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/Herring.pdf" sizes="200px"></source> <img src="/assets/img/publication_preview/Herring.pdf" class="preview z-depth-1 rounded" width="100%" height="auto" alt="Herring.pdf" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="hechler_pacific_2023" class="col-sm-8"> <div class="title">Pacific herring spawns transfer energy to coastal ecosystems</div> <div class="author"> <em>Robert M. Hechler</em>, and Martin Krkosek </div> <div class="periodical"> <em>Frontiers in Ecology and the Environment</em>, Jun 2023 </div> <div class="periodical"> *Natural history note </div> <div class="links"> <a href="https://doi.org/10.1002/fee.2675" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="kagzi_environmental_nodate" class="col-sm-8"> <div class="title">Environmental RNA degrades more rapidly than environmental DNA across a broad range of pH conditions</div> <div class="author"> Kaushar Kagzi, <em>Robert M. Hechler</em>, Gregor F. Fussmann, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Melania E. Cristescu' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Molecular Ecology Resources</em>, Jun 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1111/1755-0998.13655" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Although the use and development of molecular biomonitoring tools based on environmental nucleic acids (eDNA and eRNA; collectively known as eNAs) have gained broad interest for the quantification of biodiversity in natural ecosystems, studies investigating the impact of site-specific physicochemical parameters on eNA-based detection methods (particularly eRNA) remain scarce. Here, we used a controlled laboratory microcosm experiment to comparatively assess the environmental degradation of eDNA and eRNA across an acid–base gradient following complete removal of the progenitor organism (Daphnia pulex). Using water samples collected over a 30-day period, eDNA and eRNA copy numbers were quantified using a droplet digital PCR (ddPCR) assay targeting the mitochondrial cytochrome c oxidase subunit I (COI) gene of D. pulex. We found that eRNA decayed more rapidly than eDNA at all pH conditions tested, with detectability—predicted by an exponential decay model—for up to 57 h (eRNA; neutral pH) and 143 days (eDNA; acidic pH) post organismal removal. Decay rates for eDNA were significantly higher in neutral and alkaline conditions than in acidic conditions, while decay rates for eRNA did not differ significantly among pH levels. Collectively, our findings provide the basis for a predictive framework assessing the persistence and degradation dynamics of eRNA and eDNA across a range of ecologically relevant pH conditions, establish the potential for eRNA to be used in spatially and temporally sensitive biomonitoring studies (as it is detectable across a range of pH levels), and may be used to inform future sampling strategies in aquatic habitats.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="hechler_crow_2021" class="col-sm-8"> <div class="title">Crow predation on intertidal invertebrates</div> <div class="author"> <em>Robert M. Hechler</em> </div> <div class="periodical"> <em>Frontiers in Ecology and the Environment</em>, Jun 2021 </div> <div class="periodical"> *Natural history note </div> <div class="links"> <a href="https://doi.org/10.1002/fee.2424" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Robert M. Hechler. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> </body> </html>